### ISSUE TITLE
Links styling

### ISSUE THREAD

**Issue by alexisbronchart** [gonzalezreal/swift-markdown-ui#109]
I'd like to be able to customise links (font, color, ...) how can I do this? `MarkdownStyle` doesn't seem to allow it.

**Comment by gonzalezreal** [comment]
At the moment, you can use the `.accentColor(_:)` modifier to change the color of the links.

**Comment by yonaskolb** [comment]
I've love to have a way to remove the underline from links. Is that possible?

**Comment by gonzalezreal** [comment]
MarkdownUI 2.0, currently under development, will support themes allowing link customization among other things.

**Comment by RickiG** [comment]
Looking forward to it, we are rewriting an app where there is html styled text from the backend being changed to markdown. We have 2 things we can't achieve currently. Using a custom font, and using different fonts for header and body. Gotta love designers:)
We are scanning the attributedString looking for traits and substituting it manually, would love to have MarkdownUI handle these cases for us too. #featurerequest

**Comment by gonzalezreal** [comment]
Solved in MarkdownUI 2.0:
https://github.com/gonzalezreal/swift-markdown-ui/releases/tag/2.0

**Comment by flopshot** [comment]
It's in the demo
```swift

.markdownTextStyle(\.link) {
    ForegroundColor(.blue)
    UnderlineStyle(.init(pattern: .solid))
}


```
